### PR TITLE
Move project references to version rc3-23729

### PR DESCRIPTION
**Pull request by iamjasonp**
Consume the latest packages to take in the System.Net Linux certificate changes

### CODE REVIEW

**Comment by iamjasonp** [comment]
@dotnet-bot test outerloop please

**Comment by iamjasonp** [comment]
@dotnet-bot test outerloop please

File: run-test.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ usage()
     echo "    --restrict-proj <regex>       Run test projects that match regex"
```
Comment (roncain) [inline run-test.sh]: FYI, I expect this script to be deprecated shortly.  But don't alter the PR on that account.

**Comment by roncain** [comment]
LGTM

File: run-test.sh (hunk 1 above)
Comment (iamjasonp) [inline run-test.sh]: We've been talking about deprecating this script for a while. In the meantime, a couple of syntax errors were causing the script to *defintely* fail :) This was purely a quick syntax fix to see if testing in CI would cooperate.

**Comment by iamjasonp** [comment]
BTW, this advance in packages hasn't *proven* to improve the tcp+certificate situation (the repro project is now hanging and still not succeeding in my testing, and I'm debugging through) 

Merge anyway? (I lean towards yes)

**Comment by roncain** [comment]
LGTM -- yes, I lean toward merging and see what it fixes

**Pull request merged**